Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -90,7 +91,7 @@ def getIds : Syntax → Array Syntax
 @[inherit_doc linter.dupNamespace]
 def dupNamespace : Linter where run := withSetOptionIn fun stx ↦ do
   if Linter.getLinterValue linter.dupNamespace (← getOptions) then
-    match getIds stx with
+    match ← getNames stx with
       | #[id] =>
         let ns := (← getScope).currNamespace
         let declName := ns ++ (if id.getKind == ``declId then id[0].getId else id.getId)
```

Comment:
These lines add the current namespace at the beginning of the output of `getNames`.  However, `getNames` already handles namespaces, so this is no longer needed: it was a "fix" for `getIds`, but now it can be removed.

Reply:
make sense, done!